packages/common/src/constraints/ScoreCandidates.ts

```diff
@@ -116,7 +117,9 @@ export const scoreCandidate = async (
       scaledDefined(candidateLocation, locationValues, 0, 1) || 0;
     const locationScore = bannedProvider
       ? 0
-      : latestCandidateLocation?.city == "No Location"
+      : latestCandidateLocation?.city == "No Location" ||
```

Comment:
If you use a string more than in one place, that string deserves to be a constant or an enum. Constant string is enough in this case I'd say